### ISSUE TITLE
Since new smartphone no UHB-DNS through NordVPN?

### ISSUE THREAD

**Issue by DigitArier** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#686]
Hi everyone,
since I changed smartphones (Samsung Galaxy A52 -> Samsung Galaxy A54) every site comes through the custom DNS at NordVPN. Attached are three screenshots from [DNSleaktest.com](https://dnsleaktest.com/).

The first without NordVPN (DNS configured in DHCP) works perfectly.
The second with NordVPN without custom DNS.
The third with NordVPN with custom DNS.
1.
![DHCP](https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/assets/145800708/9d302a57-4d59-41be-8e2a-b1ec83c394ba)
2.
![NordVPN - DNS](https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/assets/145800708/967e7c97-3be2-4f23-a499-262ae2d1a159)
3.
![NordVPN + DNS](https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/assets/145800708/d40b5df5-2a4e-4410-a6a6-b10a866701bd)

By the way, DHCP settings are in all screenshots set manually to the UHB DNS Server.

Of course, everything was working as it should before. How do I get out of this mess without giving up VPN or even rooting the phone?

Best regards
Digit

**Comment by funilrys** [comment]
Hey @DigitArier,

can you please check the output of `xxx.allover.co.za` ?

If DNS is routed to our network, it should return `0.0.0.0` (A) or `::` (AAAA).

Thank you in advance.

Cheers!
Nissar

**Comment by funilrys** [comment]
Correction: It doesn't have anything to do with the underlying changes of https://github.com/Ultimate-Hosts-Blacklist/dev-center/issues/63.

If I revert the changes against a test server, I still get the public IP instead of `0.0.0.0` or `::`. Maybe Nord is doing more that we think it does 🤔

**Comment by DigitArier** [comment]
Hi @funilrys,
thanx for the quick response. I can now say that the problem has nothing to do with the phone change, as I tested it on a computer running Win11. The problem has only existed for a month, by the way. At first I was surprised that the sponsored links were not blocked on Google, but when links from my GMX mailbox went through, I became suspicious.

Cheerz
Digit

**Comment by DigitArier** [comment]
Hi @funilrys ,

I've got an update for that issue. I was in correspondence with the NordVPN support until today (sadly they are not the smartest).
This is the final mail I can present you:

'Thank you for your reply.
 
Thank you for your patience with this issue and we apologize for the longer update on your ticket.
 
We have passed the issue to our technician's team, however, we must note that the DNS issues are already known and being fixed. 
 
However, we do not have an ETA on when the issues will be fully resolved.'

Regards
Digit
